packages/template-blank-svelte/README.md

```diff
@@ -0,0 +1,34 @@
+# NativeScript Core with Svelte Blank Template
+App templates help you jump start your native cross-platform apps with built-in UI elements and best practices. Save time writing boilerplate code over and over again when you create new apps.
+
+## Quick Start
+Execute the following command to create an app from this template:
+
+```
+ns create my-blank-svelte --template @nativescript/template-blank-svelte
```

Comment:
if https://github.com/NativeScript/nativescript-cli/pull/5418 is merged, will this become `ns create my-blank-svelte --svelte`

Reply:
Ah, nevermind this is the docs for the template itself